SharePic/app/src/main/java/edu/sdsu/cs/sharepic/activity/SelectProfileActivity.java

```diff
@@ -1,26 +1,94 @@
 package edu.sdsu.cs.sharepic.activity;
 
+import android.content.Context;
+import android.content.Intent;
 import android.os.Bundle;
-import android.support.v7.app.AppCompatActivity;
+import android.support.v7.app.ActionBarActivity;
 import android.view.Menu;
 import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.ListView;
+
+import com.oguzdev.circularfloatingactionmenu.library.FloatingActionButton;
+import java.util.ArrayList;
+import java.util.Iterator;
 
 import edu.sdsu.cs.sharepic.R;
+import edu.sdsu.cs.sharepic.classes.Constants;
+import edu.sdsu.cs.sharepic.model.Account;
+import edu.sdsu.cs.sharepic.model.Profile;
+import edu.sdsu.cs.sharepic.model.Profiles;
 
+public class SelectProfileActivity extends ActionBarActivity {
 
-public class SelectProfileActivity extends AppCompatActivity {
+    ListView listView;
+    ArrayList<String> items = new ArrayList<>();
+    ArrayAdapter<String> itemsAdapter;
+    private static final int INTENT_REQUEST_CODE = 1;
+    Account[] supportedAccounts;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_select_profile);
+        setTitle(Constants.MAIN_TITLE);
+
+        supportedAccounts = Account.supportedAccounts(getApplicationContext());
+
+        listView = (ListView) findViewById(R.id.listView);
+        items = new ArrayList<>();
+        itemsAdapter =
+                new ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, items);
+        listView.setAdapter(itemsAdapter);
+        listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+            @Override
+            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                Context mainActivity = getApplicationContext();
+                Intent goDetail = new Intent(mainActivity, ProfileDetailActivity.class);
```

Comment:
goDetail, variable name does not show what it holds. A better name is profileDetailActivityIntent